### PR TITLE
LPD-38866 Change createBasicArticleWithFriendlyURL() to fill with fillAndClickOutside

### PR DESCRIPTION
**Pull request by gergelyszalay**
Hi,
May I ask to review my PR ?

Issue: https://liferay.atlassian.net/browse/LPD-38865

Solution is to use fillAndClickOutside instead of fill.

Thank you in advance!

BR

image

### CODE REVIEW

**Comment by gergelyszalay** [comment]
ci:test:content-management-playwright

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:content-management-playwright

File: modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts [hunk 1]
```diff
@@ -168,19 +168,25 @@ export class JournalEditArticlePage {
...
+			friendlyURL
+		);
 	}
 
-	async createBasicArticleWithFriendlyURL(site, page, articleTitle?: string) {
+	async createBasicArticleWithFriendlyURL(site, structureTitle?: string) {
 		await this.journalPage.goto(site.friendlyUrlPath);
 		await this.journalPage.goToCreateArticle(
-			articleTitle || 'Basic Web Content'
+			structureTitle || 'Basic Web Content'
```
Comment (DiegoHu97) [inline modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts:181]: could we chnage this to structureName to be consistent with goToCreateArticle() method?

**Review by DiegoHu97 — CHANGES_REQUESTED** [review]
LGTM!  Just a small change :smile:

File: modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts (hunk 1 above)
Comment (gergelyszalay) [reply modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts:181]: renamed the variable, Thanks

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:content-management-playwright

**Comment by gergelyszalay** [comment]
ci:test:sf

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**